packages/trader/src/sass/app/modules/modals.scss

```diff
@@ -1,2 +1 @@
 @import '../_common/base/index';
```

Comment:
Is this file necessary anymore? Can't we just remove this file?

Reply:
@mahdipourziaei-binary this is modals.

Reply:
@Mahboobeh-binary it's just importing base styles, isn't that so?

Reply:
@mahdipourziaei-binary no its the `modals.jsx` which is a component just for trader.